Behavioral.Automation.Selenium/Behavioral.Automation/Services/ConfigServiceBase.cs

```diff
@@ -31,6 +31,8 @@ public static class ConfigServiceBase
         private const string BrowserBinaryLocationString = "BROWSER_BINARY_LOCATION";
```

Comment:
What if we will use `Behavioral.Automation.Configs.ConfigManager`?

Reply:
updated